tests/appsec/iast/sink/test_xpath_injection.py

```diff
@@ -0,0 +1,52 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+import pytest
+from utils import context, coverage, released, missing_feature
+from ..iast_fixtures import SinkFixture
+
+if context.library == "cpp":
+    pytestmark = pytest.mark.skip("not relevant")
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@released(
```

Comment:
@jandro996 We can keep this simpler by just add `java=1.18.0`.  add another decorator like `@missing_feature(weblog_variant="ratpatck", reason="Sources not supported yet")`, since it's supported by most frameworks and on the same versions.

Reply:
even simpler : 

```suggestion
@released(java={"ratpack": "?", "*": "1.18.0"})
```

Reply:
done

Reply:
done